src/dealDashboard/discussionsService.ts

```diff
@@ -423,13 +442,24 @@ export class DiscussionsService {
        * the voter address from the list of down-votes (and vice versa)
        */
       if (message[endpoints[typeInverse]].includes(this.currentWalletAddress)) {
-        const success = await this.convo.comments[typeInverse](this.currentWalletAddress, token, commentId);
-        if (!success) return false;
+        const inverseVoteResponse = await this.convo.comments[typeInverse](this.currentWalletAddress, token, commentId);
+        if (inverseVoteResponse.error) throw inverseVoteResponse.error;
+
+        if (inverseVoteResponse.success) return true;
```

Comment:
seems redundant, but I'm adding this just to make sure.
Reason: The convo space has various returns types for each response.
  In a later update, they update the types a bit